crates/bevy_pbr/src/lib.rs

```diff
@@ -149,6 +151,12 @@ impl Plugin for PbrPlugin {
             "render/parallax_mapping.wgsl",
             Shader::from_wgsl
         );
+        load_internal_asset!(
+            app,
+            VIEW_TRANSFORMATIONS_SHADER_HANDLE,
+            "render/view_transformations.wgsl",
```

Comment:
Maybe not just 'view transformations' but 'transformations'?

Reply:
No longer sure about the comment. :) Just consider it and I’ll trust what you think is best.

Reply:
I think this would make sense if we can later have mesh related transformations in this same file without issue. Currently the functions here only depend on view bindings.